frontend/src/constants/CampManagementConstants.ts

```diff
@@ -0,0 +1,45 @@
+// export const MONTHS = [
+//   "January",
+//   "February",
+//   "March",
+//   "April",
+//   "May",
+//   "June",
+//   "July",
+//   "August",
+//   "September",
+//   "October",
+//   "November",
+//   "December",
+// ];
+
+export const MONTHS: { [key: string]: number } = {
+  January: 0,
+  February: 1,
+  March: 2,
+  April: 3,
+  May: 4,
+  June: 5,
+  July: 6,
+  August: 7,
+  September: 8,
+  October: 9,
+  November: 10,
+  December: 11,
+};
+
+export const getMonthIndex = (month: string): number => MONTHS[month];
+
+export const YEARS = [
```

Comment:
if we hardcode the years, wouldn't the application technically break once we hit 2032 lol. Why not construct the years based on the current year?

Reply:
Yeah, tbh I just assumed that was OK with us but will change to current year + 10